xml_converter/src/packaging_xml.cpp

```diff
@@ -168,7 +168,7 @@ vector<Parseable*> parse_pois(rapidxml::xml_node<>* root_node, map<string, Categ
             for (size_t category_index = 0; category_index < categories.size(); category_index++) {
                 for (size_t i = 0; i < categories[category_index]->icon_attributes.size(); i++) {
```

Comment:
looks like more robust tests are needed here

Reply:
Good catch. The test seems to have only worked because the size of the two arrays was the same.